tests/lib/bin/mountinfo-tool

```diff
@@ -911,34 +913,44 @@ class RenumberMountOptionsTests(unittest.TestCase):
         """
         numbers are allocated from subsets matching the key, this reduces delta.
         """
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
-        self.assertEqual(renumber_mount_option("size=200", self.seen), "size=1")
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
+        self.assertEqual(renumber_mount_option("pipe_ino=100", self.seen), "pipe_ino=0")
+        self.assertEqual(renumber_mount_option("pipe_ino=100", self.seen), "pipe_ino=0")
+        self.assertEqual(renumber_mount_option("pipe_ino=200", self.seen), "pipe_ino=1")
+        self.assertEqual(renumber_mount_option("pipe_ino=100", self.seen), "pipe_ino=0")
         self.assertEqual(renumber_mount_option("fd=21", self.seen), "fd=0")
         self.assertEqual(renumber_mount_option("fd=21", self.seen), "fd=0")
         self.assertEqual(renumber_mount_option("fd=45", self.seen), "fd=1")
         self.assertEqual(renumber_mount_option("fd=21", self.seen), "fd=0")
 
+    def test_size_not_allocated(self):
```

Comment:
I think a more clear name for this function would be `test_renumber_size_variable` because the function currently always rewrites size to be variable and it's not clear what allocation refers to in this context if you're just looking at this function and not at the history or the function right above it.

```suggestion
    def test_renumber_size_variable(self):
```